### DISCUSSION TITLE
ROI setting

### DISCUSSION THREAD

**Discussion by heruoyu1206** [ifm/ifm3d#453] [Q&A]
Hello, 
Is it possible to set ROI for ifm O3xxx cameras? If yes, how? 
For O3D, the frame rate is limited by the filter application, if ROI can be set before the data processing, then the frame rate could be higher. 
Thanks！

**Comment by heruoyu1206** [comment]
Just a reminder.

**Comment by lola-masson** [comment]
Hi @heruoyu1206,

Unfortunately, this is not something available for any of the O3 cameras. For the O3D, you can use the downsampled 23k resolution, but this is for the whole image, not just a region of interest.
